requirements_test.txt

```
@@ -10,5 +10,7 @@ PyYAML>=3.11
 requests >= 2.24.0
 responses
 retry
-scikit-learn==0.24.2
+# Note: keep it as this file is installed in quantum, which is Py3.7
+scikit-learn==1.0.2; python_version=="3.7"
+scikit-learn==1.0.2; python_version>="3.8"
```

These are the same version. Did you instead mean to do the following?
```suggestion
scikit-learn==0.24.2; python_version=="3.7"
scikit-learn==1.0.2; python_version>="3.8"
```

---

No,
I meant that once we bump scikit-learn to 1.1.X
we need to keep
scikit-learn==1.0.2; python_version=="3.7"